Template/LO/Cards/run_card.dat

```diff
@@ -79,11 +79,12 @@ $ckkw
 			     ! 2 is new strategy (using only the denominator)
 $psoptim			     
 #*********************************************************************
-# Generation bias, check the wiki page below for more information:   *
+# Generation bias and customization (custom cuts/scale/...)          *
 #  'cp3.irmp.ucl.ac.be/projects/madgraph/wiki/LOEventGenerationBias' *
 #*********************************************************************
  %(bias_module)s = bias_module  ! Bias type of bias, [None, ptj_bias, -custom_folder-]
  %(bias_parameters)s = bias_parameters ! Specifies the parameters of the module.
+ %(custom_fcts)s = custom_fcts ! List of files containing user hook function
```

Comment:
pass bias modules as hidden and change dummy model to call a function in the dummy file